plugins/binanceExchange_ws.go

```diff
@@ -140,12 +220,14 @@ func makeMapEvents() *mapEvents {
 type events struct {
 	SymbolStats *mapEvents
 	BookStats   *mapEvents
+	OrderEvents *mapEvents
```

Comment:
Why is this called `OrderEvents`?

I'm confused because we are fetching the User Stream and getting Trade History, so was expecting this to be called either `TradeHistoryEvents` or `UserEvents`